src/app/remotes/user-roles-permissions/user-roles-permissions.component.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing'
-import { NO_ERRORS_SCHEMA } from '@angular/core'
+import { ElementRef, NO_ERRORS_SCHEMA } from '@angular/core'
```

Comment:
imports from @angular/core should be the first import in the file

Reply:
done